operations/network/management_network/bican_technical_details.md

```diff
@@ -0,0 +1,252 @@
+
+## 3 Management CAN(CMN)
+
+The original CAN released in Shasta 1.1 contained the ability to access NCN workers, masters and storage directly via SSH for administrative purposes.
+This administrative traffic was co-mingled with general user traffic for jobs.
+Based on customer requests a new mechanism for administrative access to workers, masters and storage nodes will be added in CSM 1.2.
+The new Customer Management Network (CMN) will be created as **a separate-and-distinct VLAN and Subnet on the Management Network** and uplink at the edge to the Customer network.
+This new CMN network will allow SSH into the NCNs and CAN access will be disallowed.
+NOTE this is generally for ingress access for administrative purposes.
+
+### 3.1 Traffic Separation and Routing
+
+Enabling the CMN at installation time will have the following effects:
+
+- Adding uplinks to the Customer Site similar to the original CAN.
+- Creating of a new VLAN on the Management Network.
+- Adding a new subnet and routing to the Management Network and NCNs.
+
+### 3.2 Endpoint Naming
+
+If enabled during installation a ".CMN" suffix will be generated by CSI and used in SLS and Authoritative DNS.
+The "plumbing" of this will occur as previously described in the CHN.
+
+Examples:
+
+- SSH `ncn-w001.cmn.tld`
+
+### 3.3 Endpoint Addressing
+
+For the CSM 1.2 release the CMN will only be available via customer-supplied IPv4 addressing.
+
+### 3.4 Changes
+
+#### 3.4.1 Touchpoints: effects and changes
+
+- CMN required beginning with this release.
+  - Customer will supply a subnet similar to the way the CAN is deployed.
+  Sizing is Number of NCNs plus a couple more addresses (TBD).
+- Edge access to the CMN will need to be configured with the customer site.
+  - ACL development.
+- The Management network will require the following changes:
+  - Addition of the new CMN VLAN.  This should be similar to the existing CAN configuration.
+  - Termination of the new CMN VLAN on ports supporting NCNs.
+  - Addition of Customer-supplied CMN IP&#39;s to the management switches to support routing.
+- NCN workers, masters and storage will require the following changes:
+  - Image support for CMN VLAN, addressing and routing.
+- CSI changes to support the new network and naming.
+
+#### 3.4.2 When configuration occurs
+
+- During installation as part of CSI data generation.
+- During installation as part of Management Network configuration.
+- During installation as part of NCN deployment.
+
+#### 3.4.3 Ability to change post install
+
+- NOT RECOMMENDED and would be manual.
+
+## 4 CAN External/Site Access (site DNS, LDAP, etc...)
+
+System access to site or external resources, like the Internet, site DNS and site LDAP were previously provided over the CAN.
+By default this CAN access path will remain, but for the CSM 1.2 release it will be possible during installation to select system-to-site access over the CHN or CMN.
```

Comment:
```suggestion
By default this CAN access path remains, but for the CSM 1.2 release it is possible during installation to select system-to-site access over the CHN or CMN.
```